contributors/design-proposals/downward_api_resources_limits_requests.md

```diff
@@ -1,4 +1,4 @@
-# Downward API for resource limits and requests
+# Downward API for resources limits and requests
```

Comment:
it should likely always be "resource limits" - the circumstances in which "resources limits" would be appropriate are rare